manifests/local/networking-modules.yaml

```diff
@@ -3,4 +3,4 @@ path: modules/network/basic-cdk/
 targetAccount: primary
 parameters:
   - name: internet-accessible
-    value: true
+    value: false
```

Comment:
why?

Reply:
For testing, i was using a manifest file with and without internet. I can flip this back